cns/api.go

```diff
@@ -212,3 +213,12 @@ type DeleteHostNCApipaEndpointRequest struct {
 type DeleteHostNCApipaEndpointResponse struct {
 	Response Response
 }
+
+type NmAgentSupportedApisRequest struct {
+	GetNmAgentSupportedApisURL string
```

Comment:
Curious, who will be making this request? Why do they need to pass a URL?

Reply:
DNC will be making this request.

Reply:
Curious if DNC will making this request, why the code is in CNS side? Does that mean CNS will decode the response?